modules/cnf-rfhe-notifications-api-refererence.adoc

```diff
@@ -0,0 +1,321 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+:_content-type: REFERENCE
+[id="cnf-rfhe-notifications-api-refererence_{context}"]
+= Redfish hardware event notifications REST API reference
+
+You can use the Redfish hardware fast event notifications REST API to subscribe an application to Redfish hardware events.
+
+Subscribe distributed unit (DU) applications to Redfish hardware events using the resource address `/cluster/node/<node_name>/redfish/event`, where `<node_name>` is the cluster node where Redfish compliant hardware is located.
```

Comment:
I would remove DU here and make it more generic.
`Subscribe applications to ...`